superset-frontend/src/views/components/Menu.tsx

```diff
@@ -185,6 +185,24 @@ export function Menu({
   const screens = useBreakpoint();
   const uiConig = useUiConfig();
 
+  const nonItemsToRender = {
+    'Upload a CSV': 'Upload a CSV',
```

Comment:
instead of doing this just comment out the childs that we don't want to render, and remove the if condition in `createPlusMenuItems`